python/ray/scripts/scripts.py

```diff
@@ -430,6 +430,12 @@ def debug(address):
     default=None,
     help="the port for dashboard agents to listen for grpc on.",
 )
+@click.option(
+    "--dashboard-grpc-port",
```

Comment:
ideally, I think `--dashboard-rpc-port` is better given grpc is just implementation details but since we already have `--dashboard-agent-grpc-port`, I think it's fine to name this way.